src/Yaapii.Atoms/Text/SubText.cs

```diff
@@ -43,11 +43,11 @@ public SubText(String text, int strt) : this(new LiveText(text), strt)
         /// </summary>
         /// <param name="text">text to extract from</param>
         /// <param name="strt">where to start</param>
-        /// <param name="end">where to end</param>
-        public SubText(String text, int strt, int end) : this(
+        /// <param name="len">length of substring</param>
+        public SubText(String text, int strt, int len) : this(
```

Comment:
Please adjust the parameter names so we can see for what the parameter is used. So you can remove the parameter description from header. Most programmers forget to adjust these comments and the parametrer name should say what the param stands for. 

Also please don't use abbreviations. I think this is old code and we should avoid something like that